<?xml version="1.0"?>

<launch>
    <node name="CreateFile" pkg="mbzirc" type="createfileworld" output="screen" args="$(find mbzirc)"/>
	
    <!--node name="Waypoints_File" pkg="mbzirc" type="waypoint_publisher" output="screen"/-->
	
	<include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find mbzirc)/world/generate_world.world"/>
    	<arg name="paused" value="true"/>
  	</include>
  	
    <include file="$(find grvc_quadrotor_hal)/launch/spawn_quad_gazebo.launch">
        <arg name="robot_name" value="quad1"/>
        <arg name="pos_x" value="65"/>
        <arg name="pos_y" value="-25"/>
        <arg name="pos_z" value="0.2"/>
    </include>

    <include file="$(find grvc_quadrotor_hal)/launch/spawn_quad_gazebo.launch">
        <arg name="robot_name" value="quad2"/>
        <arg name="pos_x" value="65"/>
        <arg name="pos_y" value="-23.5"/>
        <arg name="pos_z" value="0.2"/>
    </include>
  	
    <include file="$(find grvc_quadrotor_hal)/launch/spawn_quad_gazebo.launch">
        <arg name="robot_name" value="quad3"/>
        <arg name="pos_z" value="0.2"/>
        <arg name="pos_x" value="65"/>
        <arg name="pos_y" value="-26.5"/>
    </include>


  	<node name="Waypoints" pkg="mbzirc" type="pruebamover" output="screen"/>

	
</launch>
